2021/46xxx/CVE-2021-46388.json

```diff
@@ -2,7 +2,7 @@
     "CVE_data_meta": {
```

Comment:
{} is not allowed for {'vendor': {'vendor_data': [{'product': {'product_data': [{'product_name': 'WAGO 750-8212 PFC200 G2 2ETH RS', 'version': {'version_data': [{'version_value': 'Frimware version: 03.05.10(17)'}]}}]}, 'vendor_name': 'Wago'}]}}